MyPlayground.playground/Contents.swift

```diff
@@ -0,0 +1,63 @@
+func arithmeticsFunction(array: [Int]) -> (arithmetic: Double, mediana: Double, moda: Set<Int>) {
```

Comment:
`array` is a bad name. I know it's array. For me it's more important to know the semantics.

```suggestion
func arithmeticsFunction(numbers: [Int]) -> (arithmetic: Double, mediana: Double, moda: Set<Int>) {
```

Reply:
Fixed.